tests/Aspire.Hosting.Tests/Dashboard/DashboardServiceTests.cs

```diff
@@ -87,20 +87,14 @@ public async Task WatchResources_ResourceHasCommands_CommandsSentWithResponse()
             executeCommand: c => Task.FromResult(CommandResults.Success()),
             updateState: c => ApplicationModel.ResourceCommandState.Enabled,
             displayDescription: "Display description!",
-            parameter: new [] {"One", "Two"},
+            parameter: new[] { "One", "Two" },
             confirmationMessage: "Confirmation message!",
             iconName: "Icon name!",
             iconVariant: ApplicationModel.IconVariant.Filled,
             isHighlighted: true);
 
-        await resourceNotificationService.PublishUpdateAsync(testResource, s =>
```

Comment:
diff is messed up, look at side by side

Reply:
What do you mean? Is this just whitespace change? Note that it will conflict with a change I've just made here: https://github.com/dotnet/aspire/pull/6366

Reply:
No, a new test has been added in that placement - we can fix the merge conflict